1-org/modules/network/versions.tf

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+terraform {
+  required_version = ">= 0.13"
```

Comment:
Since the modules uses the 'optional' attribute, the required version should be >= 1.3 or > 0.14 with experiment mode on.

Reply:
This change will be done on other branch/PR.

Reply:
@Samir-Cit Amanda is right, the version bump should be added in this PR as the use of `optional` constitutes a breaking change.

Reply:
@gtsorbo good to mention, I did the terraform version bump on this branch.
Now it's using version **>= 1.3**